tsconfig.json

```diff
@@ -12,9 +12,19 @@
       "inlineSources": true,
       "outDir": "dist",
       "baseUrl": ".",
-      "skipLibCheck": true
+      "skipLibCheck": true,
+      "sourceRoot": ".",
+      "noUnusedLocals": true,
+      "paths": {
+        "denali-cli": [ "lib/index.ts" ]
```

Comment:
It's probably not...I saw the same thing in Denali and just copied it over, I'm not exactly sure what it does. I'll try removing it and see if I run into any issues.